kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/storage/AwsS3DirectoryLister.scala

```diff
@@ -77,8 +78,9 @@ object AwsS3DirectoryLister {
       .maxKeys(1000)
       .bucket(bucketAndPrefix.bucket)
       .delimiter("/")
-    bucketAndPrefix.prefix.map(addTrailingSlash).foreach(builder.prefix)
-    builder.build()
+    bucketAndPrefix.prefix.foreach(builder.prefix)
+    val request = builder.build()
```

Comment:
minor, but no need for the 'request' variable as you can just have `builder.build()` as the last line.